src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -1445,7 +1445,7 @@ public async Task RudeEdits_DocumentOutOfSync(bool breakMode)
         {
             var source0 = "class C1 { void M() { System.Console.WriteLine(0); } }";
             var source1 = "class C1 { void M() { System.Console.WriteLine(1); } }";
-            var source2 = "class C1 { void RenamedMethod() { System.Console.WriteLine(1); } }";
+            var source2 = "class C1 { int M() { System.Console.WriteLine(1); } }";
```

Comment:
>int [](http://example.com/codeflow?start=38&length=3)

Maybe use "add generic type parameter" rude edit here as well, in anticipation of changing member signatures being allowed in near future.